mscore/musescore.cpp

```diff
@@ -576,13 +621,17 @@ MuseScore::MuseScore()
       setObjectName("MuseScore");
       _sstate = STATE_INIT;
       setWindowTitle(QString(MUSESCORE_NAME_VERSION));
-      setIconSize(QSize(preferences.iconWidth * guiScaling, preferences.iconHeight * guiScaling));
+      setIconSize(QSize(preferences.getInt(PREF_UI_THEME_ICONWIDTH) * guiScaling, preferences.getInt(PREF_UI_THEME_ICONHEIGHT) * guiScaling));
 
       ucheck = new UpdateChecker();
 
       setAcceptDrops(true);
       setFocusPolicy(Qt::NoFocus);
 
+#ifdef SCRIPT_INTERFACE
```

Comment:
pluginList is moved from preferences to PluginManager so there needs to be a pluginManager available from start of application